staging/kos/pkg/registry/network/subnet/strategy.go

```diff
@@ -132,13 +133,14 @@ func (ss *subnetStrategy) ValidateUpdate(ctx context.Context, obj, old runtime.O
 }
 
 func (ss *subnetStrategy) validate(s *network.Subnet) field.ErrorList {
+	glog.V(2).Infof("Validating subnet %s/%s", s.Namespace, s.Name)
```

Comment:
Here is an outline of the way I tend to use V levels.

1. Very prominent stuff that happens once per process run.
2. Less prominent stuff that happens once per process run.
3. Very prominent stuff that typically happens O(1) in an object lifecycle.
4. Less prominent stuff that typically happens O(1) in an object lifecycle.
5. Debugging details behind a computation that typically happens O(1) in an object lifecycle.
6. Stuff that typically happens multiple times in an object lifecycle.

There is a V level convention doc somewhere that I can not find right now, it talks about levels 1--4, and what I wrote above is my more precise take on that.  Higher levels than 4 _do_ get used in Kubernetes.

Reply:
Thanks for sharing your heuristic, that is very useful, so far I set log levels w/o consistent rules , on the spur of what seemed reasonable. Is [this](https://github.com/kubernetes/kubernetes/blob/release-1.1/docs/devel/logging.md) the doc you were referring to? Also, I'm not sure from the heuristic what log level you're suggesting the commented line of code. ea5703a changes the verbosity of the statement your comment is about to 4.

Reply:
Yes, that looks like the doc I was thinking of.